tendrl/node_agent/message/handler.py

```diff
@@ -57,7 +58,13 @@ def bind_unix_listener(self):
             self.sock.setblocking(0)
             self.sock.bind(socket_path)
             self.sock.listen(50)
-        except (TypeError, BlockingIOError, socket_error, ValueError) as ex:
+            gid = pwd.getpwnam("tendrl-user").pw_gid
```

Comment:
@nthomas-redhat here i am giving permission for that user and entire group is wrong, i have to give permission to that user only like os.chown(socket_path, uid, -1), -1 is used to leave one of the ids unchanged.